src/main/kotlin/org/taktik/icure/asyncdao/impl/PatientDAOImpl.kt

```diff
@@ -264,8 +264,8 @@ class PatientDAOImpl(
 		emitAll(client.queryView<Array<String>, String>(viewQuery).mapNotNull { it.value })
 	}
 
-	@View(name = "by_hcparty_address", map = "classpath:js/patient/By_hcparty_address.js")
-	override fun listPatientIdsByHcPartyAndAddress(searchString: String?, healthcarePartyId: String): Flow<String> = flow {
+	@View(name = "by_hcparty_address_postalcode_housenumber", map = "classpath:js/patient/By_hcparty_address_postalcode_housenumber.js")
```

Comment:
Be aware that: 
- Creating a new view requires re-indexing all databases to have access to this new view. 
- What if other clients are using this by_hcparty_address view ? Please, create another method in the DAO, in order to let us stay compatible with previous versions / with other clients. We'll decide when and how to delete them.
Plus, both of them can totally be compatible ! 
=> If user is providing postalCode or postalCode and houseNumber, use your new view. Otherwise, use the previous one. 
=> Still, make your new view compatible with the previous one, so that we can delete it later.